core/src/test/java/com/novoda/merlin/service/PingShould.java

```diff
@@ -23,66 +21,54 @@
 
     private static final String HOST_ADDRESS = "any host address";
 
-    @RunWith(Parameterized.class)
     public static class GivenSuccessfulRequest {
 
-        @Parameterized.Parameters(name = "{0}")
-        public static Collection<Integer[]> data() {
-            return ResponseCode.toParameterList();
-        }
-
-        private final Ping ping;
+        private Ping ping;
 
         @Mock
-        private HostPinger.ResponseCodeFetcher mockResponseCodeFetcher;
+        private ResponseCodeFetcher mockResponseCodeFetcher;
+        @Mock
+        private ResponseCodeValidator mockResponseCodeValidator;
 
-        private final int responseCode;
+        private final int responseCode = 234;
 
-        public GivenSuccessfulRequest(int responseCode) {
+        @Before
+        public void setup() {
```

Comment:
It's just the only place where it's lowercased. I'd prefer it lowercased but I suppose consistency has precedence